projects/marketing-site/pages/funding.tsx

```diff
@@ -10,19 +10,19 @@ import {
 const LandingPage = () => {
   return (
     <Layout>
-      <div className="isolate bg-black">
+      <div className="isolate dark:bg-black bg-white">
         <BlueSplash />
         <SimpleNavbar />
         <RedSplash height="sm" />
       </div>
       <main>
-        <div className="relative px-6 lg:px-8">
+        <div className="px-6 lg:px-8">
           <div className="mx-auto max-w-3xl pt-20 pb-32 sm:pt-48 sm:pb-40">
             <div>
               <h1 className="text-4xl font-bold tracking-tight sm:text-center sm:text-6xl">
                 L2DAO Proposal
               </h1>
```

Comment:
we need to change the color for light mode here. you can't see the text with light mode selected